e2e/resources/workbench-config.ts

```diff
@@ -1,5 +1,5 @@
 require('dotenv').config();
-const _ = require('lodash');
+const lodash = require('lodash');
```

Comment:
Existing issue: the rest of the codebase uses lodash/fp, our convention is to alias this as "fp". May want to reconcile this later